packages/website/plugins/generated-rule-docs/insertions/insertRuleOptions.ts

```diff
@@ -43,7 +43,7 @@ export function insertRuleOptions(page: RuleDocsPage): void {
       if (!page.rule.meta.docs.extendsBaseRule) {
         throw new Error(`Missing description for option ${optionName}.`);
       }
-      return;
+      continue;
```

Comment:
This is a slightly larger change, please let me know if I should open a separate issue to discuss this.

---

To my understanding, currently, as soon as an extension rule has an option without description, it would stop replacing the `/* insert option description */` comment with option descriptions (even if the next option may have description).

This affects `no-magic-numbers`, which some of its option are missing description (inherited ones), but some do have description ([the ones added by the extension rule](https://github.com/typescript-eslint/typescript-eslint/blob/a94004c84d96b413ea92124cc3b120d5fa217adb/packages/eslint-plugin/src/rules/no-magic-numbers.ts#L27-L44)).